series/series.go

```diff
@@ -647,3 +651,64 @@ type indexedElements []indexedElement
 func (e indexedElements) Len() int           { return len(e) }
 func (e indexedElements) Less(i, j int) bool { return e[i].element.Less(e[j].element) }
 func (e indexedElements) Swap(i, j int)      { e[i], e[j] = e[j], e[i] }
+
+// StdDev calculates the standard deviation of a series
+func (s Series) StdDev() float64 {
+	stdDev := stat.StdDev(s.Float(), nil)
+	return stdDev
+}
+
+// Mean calculates the average value of a series
+func (s Series) Mean() float64 {
+	stdDev := stat.Mean(s.Float(), nil)
+	return stdDev
+}
+
+// Max return the biggest element in the series
+func (s Series) Max() Element {
+	if s.elements.Len() == 0 {
+		return nil
+	}
+	max := s.elements.Elem(0)
+	for i := 0; i < s.elements.Len(); i++ {
+		elem := s.elements.Elem(i)
+		if elem.Greater(max) {
+			max = elem
+		}
+	}
+	return max
+}
+
+// Min return the lowest element in the series
+func (s Series) Min() Element {
+	if s.elements.Len() == 0 {
+		return nil
```

Comment:
Returning `nil` here is problematic. If you add a test where you have an empty series, it will panic. This should either return an initialized empty element of the appropriate type or we could make it so that it will return `float64` instead of `Element` as we discussed in the past.